### PR TITLE
261 -2 Youglish Teacher Frontend

### PR DESCRIPTION
**Pull request by dpmacisaac**
…ate-youglish-rest-api-harness

### CODE REVIEW

File: ComSemApp/templates/ComSemApp/teacher/expression_form.html [hunk 1]
```diff
@@ -97,11 +116,11 @@ <h4 class="text-muted m-b-10 font-14">
         $(function(){
             $('#recordingslist').html("<audio controls id='audioElement' src='" + "{{ object.audio.url }}" + "' type='audio/mpeg'></audio>");
             $('#deleteRecordingButton').show();
-        })
+        });
```
Comment (dpmacisaac) [inline ComSemApp/templates/ComSemApp/teacher/expression_form.html:119]: love the old code issues in this code base

File: ComSemApp/templates/ComSemApp/teacher/expression_form.html [hunk 2]
```diff
@@ -79,6 +77,27 @@ <h4 class="text-muted m-b-10 font-14">
...
+                        <input class=" form-control rounded-end m-1" type="text" id="wordInput" placeholder="Enter a word">
+                        <button class="btn btn-primary m-1" type="button" id="addButton">+</button>
+                    </div>
+                    <div class="d-flex flex-wrap" id="bubbleContainer"></div>
+                </div>
+                <style>
+                    .btn-bubble:hover {
+                        background-color: #DB1A1A;
+                        border-color: #DB1A1A; 
+                      }
```
Comment (dpmacisaac) [inline ComSemApp/templates/ComSemApp/teacher/expression_form.html:95]: There is basically no good bootstrap way to do this from my research

File: .gitignore [hunk 3]
```diff
@@ -12,6 +12,7 @@ app-messages/*
 
 # dev database
 db.sqlite3
+.vscode/*
```
Comment (aedmondson) [inline .gitignore:15]: Doesn't really matter, but you can just say `.vscode/` (no star)

File: ComSemApp/templates/ComSemApp/teacher/expression_form.html (hunk 2 above)
Comment (aedmondson) [reply ComSemApp/templates/ComSemApp/teacher/expression_form.html:95]: yeah... bootstrap is good at everything except the things it's bad at

File: ComSemApp/templates/ComSemApp/teacher/expression_form.html (hunk 1 above)
Comment (aedmondson) [reply ComSemApp/templates/ComSemApp/teacher/expression_form.html:119]: <3 xD

File: ComSemApp/templates/ComSemApp/teacher/expression_form.html [hunk 4]
```diff
@@ -142,11 +192,14 @@ <h4 class="text-muted m-b-10 font-14">
     if (student_id != "0"){
         worksheetFormData.append( 'student', student_id );
     }
+
```
Comment (aedmondson) [inline ComSemApp/templates/ComSemApp/teacher/expression_form.html:195]: Fine for now, but try to limit whitespace to one line at a time

**Review by aedmondson — APPROVED** [review]

**Pull request merged**